src/VisualStudio/Core/Def/ErrorReporting/VisualStudioErrorReportingService.cs

```diff
@@ -29,13 +29,16 @@ internal partial class VisualStudioErrorReportingService : IErrorReportingServic
     public VisualStudioErrorReportingService(
         IThreadingContext threadingContext,
         IVsService<SVsActivityLog, IVsActivityLog> activityLog,
-        IAsynchronousOperationListenerProvider listenerProvider,
-        SVsServiceProvider serviceProvider)
+        IVsService<SVsInfoBarUIFactory, IVsInfoBarUIFactory> vsInfoBarUIFactory,
+        IVsService<SVsShell, IVsShell> vsShell,
+        IAsynchronousOperationListenerProvider listenerProvider)
     {
         _threadingContext = threadingContext;
         _activityLog = activityLog;
         _listener = listenerProvider.GetListener(FeatureAttribute.Workspace);
-        _infoBar = new VisualStudioInfoBar(threadingContext, serviceProvider, listenerProvider);
+
+        // Attach this info bar to the global shell location for info-bars (independent of any particular window).
+        _infoBar = new VisualStudioInfoBar(threadingContext, vsInfoBarUIFactory, vsShell, listenerProvider, windowFrame: null);
```

Comment:
part of https://github.com/dotnet/roslyn/pull/72765.  Will be gone when that PR goes in.